### DISCUSSION TITLE
Auto portals and mounted portal gun

### DISCUSSION THREAD

**Discussion by Stnkybo** [BEEmod/BEE2-items#3773] [Tech Support / Q&A] [answered]
Has anyone been able to get these to work on bee2 4.39.2? Cuz I can’t

**Comment by FrozenDragon0** [comment, marked as answer]
Already known issues.

**Reply by Stnkybo** [reply]
Cheers, how long does it usually take to patch a bug like this?

**Reply by FrozenDragon0** [reply]
There is no specified date for beemod releases so fixes could take months before being released.
